extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerBuildSupport.java

```diff
@@ -81,6 +84,16 @@ public void updateClasspath(IProject project, IProgressMonitor monitor) throws C
             List<IClasspathEntry> sourceEntries = getSourceEntries(rootPath, project, sourcesResult, outputFullPath, isTest, monitor);
             classpath.addAll(sourceEntries);
 
+            String resourceOutputUri = getOutputUriByKind(outputResult.getItems(), "resource");
+            IPath resourceOutputFullPath = getOutputFullPath(resourceOutputUri, project);
+            // resource output is nullable according to Gradle API definition.
+            if (resourceOutputFullPath != null) {
```

Comment:
The Line 78 should use similar logic such as `if (outputFullPath != null) { ... }`.